### ISSUE TITLE
java.util.HashSet cannot be cast to java.util.ArrayList

### ISSUE THREAD

**Issue by TakoidGit** [bloodmc/GriefDefender#35]
```
[16:46:45] [Server thread/WARN]: [GriefDefender] Task #18 for GriefDefender v1.1.0 generated an exception
java.lang.ClassCastException: java.util.HashSet cannot be cast to java.util.ArrayList
	at com.griefdefender.task.TaxApplyTask.run(TaxApplyTask.java:76) ~[?:?]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.14.4.jar:git-Paper-159]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1147) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-159]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
[16:46:45] [Server thread/WARN]: [GriefDefender] Task #19 for GriefDefender v1.1.0 generated an exception
java.lang.ClassCastException: java.util.HashSet cannot be cast to java.util.ArrayList
	at com.griefdefender.task.TaxApplyTask.run(TaxApplyTask.java:76) ~[?:?]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.14.4.jar:git-Paper-159]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1147) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-159]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
[16:46:45] [Server thread/WARN]: [GriefDefender] Task #20 for GriefDefender v1.1.0 generated an exception
java.lang.ClassCastException: java.util.HashSet cannot be cast to java.util.ArrayList
	at com.griefdefender.task.TaxApplyTask.run(TaxApplyTask.java:76) ~[?:?]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.14.4.jar:git-Paper-159]
	at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1147) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-159]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-159]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```
Seems to be related to the tax system. This is on dev build v1.1.1-DEV2.

Came up once and hasn't come up since.

**Comment by TakoidGit** [comment]
```
[11:34:11 WARN]: [GriefDefender] Task #18 for GriefDefender v1.1.0 generated an exception
java.lang.ClassCastException: java.util.HashSet cannot be cast to java.util.ArrayList
        at com.griefdefender.task.TaxApplyTask.run(TaxApplyTask.java:76) ~[?:?]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.14.4.jar:git-Paper-159]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1147) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-159]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
[11:34:11 WARN]: [GriefDefender] Task #19 for GriefDefender v1.1.0 generated an exception
java.lang.ClassCastException: java.util.HashSet cannot be cast to java.util.ArrayList
        at com.griefdefender.task.TaxApplyTask.run(TaxApplyTask.java:76) ~[?:?]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.14.4.jar:git-Paper-159]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1147) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-159]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
[11:34:11 WARN]: [GriefDefender] Task #20 for GriefDefender v1.1.0 generated an exception
java.lang.ClassCastException: java.util.HashSet cannot be cast to java.util.ArrayList
        at com.griefdefender.task.TaxApplyTask.run(TaxApplyTask.java:76) ~[?:?]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.14.4.jar:git-Paper-159]
        at org.bukkit.craftbukkit.v1_14_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:452) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.b(MinecraftServer.java:1147) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.DedicatedServer.b(DedicatedServer.java:417) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:1074) ~[patched_1.14.4.jar:git-Paper-159]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:918) ~[patched_1.14.4.jar:git-Paper-159]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```

Happening on DEV3 as well.

**Comment by bloodmc** [comment]
Fixed in 1.2.0 release.
